helpers/cuentasContablesHelper/cuentasContables.helper.go

```diff
@@ -172,10 +173,14 @@ func AsientoContable(totales map[int]float64, tipomvto string, descripcionMovto
 	transaccion.Etiquetas = string(jsonString)
 	transaccion.FechaTransaccion = time_bogota.Tiempo_bogota()
 
+	tercerodebito := true
+	tercerocredito := true
 	for clave, _ := range totales {
+		logs.Debug("******** La clave ", clave)
```

Comment:
Comentar
```suggestion
		// logs.Debug("******** La clave ", clave)
```

o Eliminar
```suggestion
```